snow/engine/avalanche/bootstrap/bootstrapper_test.go

```diff
@@ -620,111 +636,3 @@ func TestBootstrapperIncompleteAncestors(t *testing.T) {
 	require.Equal(choices.Accepted, vtx1.Status())
 	require.Equal(choices.Accepted, vtx2.Status())
 }
-
-func TestBootstrapperUnexpectedVertex(t *testing.T) {
-	require := require.New(t)
-
-	config, peerID, sender, manager, vm := newConfig(t)
-
-	vtxID0 := ids.Empty.Prefix(0)
-	vtxID1 := ids.Empty.Prefix(1)
-
-	vtxBytes0 := []byte{0}
-	vtxBytes1 := []byte{1}
-
-	vtx0 := &avalanche.TestVertex{
-		TestDecidable: choices.TestDecidable{
-			IDV:     vtxID0,
-			StatusV: choices.Unknown,
-		},
-		HeightV: 0,
-		BytesV:  vtxBytes0,
-	}
-	vtx1 := &avalanche.TestVertex{ // vtx1 is the stop vertex
-		TestDecidable: choices.TestDecidable{
-			IDV:     vtxID1,
-			StatusV: choices.Unknown,
-		},
-		ParentsV: []avalanche.Vertex{vtx0},
-		HeightV:  1,
-		BytesV:   vtxBytes1,
-	}
-
-	config.StopVertexID = vtxID1
-	bs, err := New(
-		config,
-		func(context.Context, uint32) error {
-			config.Ctx.State.Set(snow.EngineState{
-				Type:  p2p.EngineType_ENGINE_TYPE_AVALANCHE,
-				State: snow.NormalOp,
-			})
-			return nil
-		},
-	)
-	require.NoError(err)
-
-	parsedVtx0 := false
-	parsedVtx1 := false
-	manager.GetVtxF = func(_ context.Context, vtxID ids.ID) (avalanche.Vertex, error) {
-		switch vtxID {
-		case vtxID0:
-			if parsedVtx0 {
-				return vtx0, nil
-			}
-			return nil, errUnknownVertex
-		case vtxID1:
-			if parsedVtx1 {
-				return vtx1, nil
-			}
-			return nil, errUnknownVertex
-		default:
-			require.FailNow(errUnknownVertex.Error())
-			return nil, errUnknownVertex
-		}
-	}
-	manager.ParseVtxF = func(_ context.Context, vtxBytes []byte) (avalanche.Vertex, error) {
-		switch {
-		case bytes.Equal(vtxBytes, vtxBytes0):
-			vtx0.StatusV = choices.Processing
-			parsedVtx0 = true
-			return vtx0, nil
-		case bytes.Equal(vtxBytes, vtxBytes1):
-			vtx1.StatusV = choices.Processing
-			parsedVtx1 = true
-			return vtx1, nil
-		default:
-			require.FailNow(errUnknownVertex.Error())
-			return nil, errUnknownVertex
-		}
-	}
-
-	requestIDs := map[ids.ID]uint32{}
-	sender.SendGetAncestorsF = func(_ context.Context, vdr ids.NodeID, reqID uint32, vtxID ids.ID) {
-		require.Equal(peerID, vdr)
-		requestIDs[vtxID] = reqID
-	}
-
-	vm.CantSetState = false
-	require.NoError(bs.Start(context.Background(), 0)) // should request vtx1
-	require.Contains(requestIDs, vtxID1)
-
-	reqID := requestIDs[vtxID1]
-	clear(requestIDs)
-	require.NoError(bs.Ancestors(context.Background(), peerID, reqID, [][]byte{vtxBytes0}))
-	require.Contains(requestIDs, vtxID1)
-
-	manager.EdgeF = func(context.Context) []ids.ID {
-		require.Equal(choices.Accepted, vtx1.Status())
-		return []ids.ID{vtxID1}
-	}
-
-	vm.LinearizeF = func(_ context.Context, stopVertexID ids.ID) error {
-		require.Equal(vtxID1, stopVertexID)
-		return nil
-	}
-
-	require.NoError(bs.Ancestors(context.Background(), peerID, reqID, [][]byte{vtxBytes1, vtxBytes0}))
```

Comment:
This was previously testing that using the (wrong) `peerID` and `reqID` would still process `vtxBytes1`. But we no longer support that... So this test just didn't make sense anymore.